fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -800,6 +800,24 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static boolean enable_http_detail_metrics = false;
 
+    /**
+     * Size of the thread pool for asynchronously processing HTTP request business logic. This pool Handles
+     * time-consuming operations such as:
+     * - Transaction stream load awaiting publish
+     * - HTTP SQL execution
+     *
+     * This is different from {@code Config.http_worker_threads_num} which manages Netty HTTP server workers
+     * mainly for handling network I/O (connections, data transfer). If the business logic is lightweight,
+     * it can be executed directly in the worker thread; otherwise, it should be processed in this async thread
+     * pool to avoid blocking the HTTP server.
+     */
+    @ConfField(mutable = true, aliases = {"max_http_sql_service_task_threads_num"})
```

Comment:
add "comment" if possible, so it will be shown in the `admin show frontend config like 'xxx'`, provide some explanation to the end user.

Reply:
fixed